database/src/prepare.ts

```diff
@@ -10,19 +10,25 @@ export enum DatabaseState {
   SAME_VERSION = 'SAME_VERSION',
 }
 
-async function connectToDatabaseServer(): Promise<Connection | null> {
-  try {
-    return await createConnection({
-      host: CONFIG.DB_HOST,
-      port: CONFIG.DB_PORT,
-      user: CONFIG.DB_USER,
-      password: CONFIG.DB_PASSWORD,
-    })
-  } catch (e) {
-    // biome-ignore lint/suspicious/noConsole: no logger present
-    console.log('could not connect to database server', e)
-    return null
+export async function connectToDatabaseServer(
+  maxRetries = 15,
+  delayMs = 500,
```

Comment:
same hardcoded values like above. they should be defined once and used as constants.